examples/swarmion-starter/nx.json

```diff
@@ -0,0 +1,77 @@
+{
+  "extends": "@nrwl/workspace/presets/npm.json",
+  "npmScope": "swarmion-starter",
```

Comment:
Can we add a comment here stating this is important for generators as well. See https://github.com/swarmion/swarmion/issues/143